app/workers/nullify_deactivated_subscribers_worker.rb

```diff
@@ -0,0 +1,16 @@
+class NullifyDeactivatedSubscribersWorker
+  include Sidekiq::Worker
+
+  def perform
+    subscribers.find_each(&:nullify!)
```

Comment:
Might just want to do this in an `update_all` though can't imagine this would see sufficient load to cause any pain?

Reply:
I liked being able to use the `nullify!` method here for clarity, and figured the load shouldn't cause any problems.

Reply:
Sure thing.